sklearn/decomposition/_dict_learning.py

```diff
@@ -1061,19 +1061,19 @@ class DictionaryLearning(SparseCodingMixin, BaseEstimator):
 
     Parameters
     ----------
-    n_components : int,
+    n_components : int, optional (default=n_features)
```

Comment:
We are not anymore including the `optional` keyword.
It will look like:

```suggestion
    n_components : int, default=n_features
```

Reply:
I changed the format to what you suggested, thanks!